CMakeLists.txt

```diff
@@ -285,7 +285,7 @@ endif()
 
 if(Qt5Core_FOUND)
   set_package_properties(Qt5Core PROPERTIES TYPE REQUIRED)
-  find_package(Qt5 NO_MODULE REQUIRED COMPONENTS Gui Network)
+  find_package(Qt5 NO_MODULE REQUIRED COMPONENTS Gui Network OpenGL)
```

Comment:
Considering we don't seem to use Qt5::OpenGL anywhere, do we really need it as a required dependency?